Frontend/android-native/app/src/driver/nammaYatriPartner/res/layout/splash.xml

```diff
@@ -2,44 +2,47 @@
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@drawable/ny_ic_splash_bg"
+    android:background="@color/black"
     android:id="@+id/splash">
 
     <FrameLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:layout_gravity="center_vertical"
         android:orientation="vertical">
-
         <ImageView
-            android:id="@+id/logo_only"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_gravity="center"
-            android:src="@drawable/ny_ic_namma_yatri_kn" />
-
-        <ImageView
-            android:layout_width="316dp"
-            android:layout_height="39dp"
+            android:layout_height="match_parent"
             android:layout_gravity="center"
-            android:layout_marginTop="70dp"
-            android:paddingHorizontal="50dp"
-            android:src="@drawable/ny_ic_namma_yatri_partner_en" />
-
-        <ProgressBar
-            android:layout_width="wrap_content"
+            android:src="@drawable/ny_ic_splash_bg_trans" />
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:orientation="vertical"
+            android:gravity="center">
+            <ImageView
+                android:layout_width="wrap_content"
+                android:layout_height="100dp"
+                android:layout_margin="40dp"
+                android:layout_gravity="center"
+                android:src="@drawable/ny_ic_namma_yatri_kn" />
```

Comment:
default screen has kannada?

Reply:
It is same name which is used before updated the image.

Reply:
bro my question is something else

Reply:
Default is english